ert-tests/editorconfig-core-handle.el

```diff
@@ -47,4 +47,10 @@
                                                             (concat fixtures
                                                                     "a.js"))
                    '((("key" . "value"))))))
+
+  ;; For checking various normal whitespace (line breaks, horizontal space, vertical space, etc.)
+  (let* ((conf (concat default-directory
+                       "ert-tests/whitespaces/example-editorconfig.txt"))
+         (handle (editorocnfig-core-handle conf)))
```

Comment:
`editorocnfig` is spelled wrong here (should be `editorconfig`) :P

Otherwise, this looks right, and seems like the correct solution to me :)

Reply:
❗️ 
You are right! Thanks XP